app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt

```diff
@@ -145,7 +145,7 @@ class CompletedStoryListActivityTest {
       onView(
         atPositionOnView(
           R.id.completed_story_list,
-          0,
+          position = 0,
           R.id.completed_story_lesson_thumbnail
```

Comment:
@anandwana001 Do we want named arguments for only `position` parameter or we should have named arguments for all the three parameters here?

Reply:
All we need for consistency and it looks good too.

Reply:
@techjd You can go ahead and do the same for the other 2 parameters also.

Reply:
@prayutsu I have changed for all three parameters . PTAL